velox/docs/functions/presto/conversion.rst

```diff
@@ -667,52 +700,79 @@ is the number of whole days in the interval, HH is then number of hours between
 From IPADDRESS
 ^^^^^^^^^^^^^^
 
-Casting from IPADDRESS to VARCHAR returns a string formatted as x.x.x.x for IPV4 formatted IPV6 addresses.
-For all other IPV6 addresses it will be formatted in compressed alternate form IPV6 defined in `RFC 4291#section-2.2 <https://datatracker.ietf.org/doc/html/rfc4291.html#section-2.2>`_
+Casting from IPADDRESS to VARCHAR returns a string formatted as x.x.x.x for IPv4 formatted IPv6 addresses.
+For all other IPv6 addresses it will be formatted in compressed alternate form IPv6 defined in `RFC 4291#section-2.2 <https://datatracker.ietf.org/doc/html/rfc4291.html#section-2.2>`_
 
-IPV4:
+IPv4:
 
 ::
 
   SELECT cast(ipaddress '1.2.3.4' as varchar); -- '1.2.3.4'
 
-IPV6:
+IPv6:
 
 ::
 
   SELECT cast(ipaddress '2001:0db8:0000:0000:0000:ff00:0042:8329' as varchar); -- '2001:db8::ff00:42:8329'
   SELECT cast(ipaddress '0:0:0:0:0:0:13.1.68.3' as varchar); -- '::13.1.68.3'
 
-IPV4 mapped IPV6:
+IPv4 mapped IPv6:
 
 ::
 
   SELECT cast(ipaddress '::ffff:ffff:ffff' as varchar); -- '255.255.255.255'
 
+From IPPREFIX
+^^^^^^^^^^^^^
+
+Casting from IPPREFIX to VARCHAR returns a string formatted as *x.x.x.x/<prefix-length>* for IPv4 formatted IPv6 addresses.
+
+For all other IPv6 addresses it will be formatted in compressed alternate form IPv6 defined in `RFC 4291#section-2.2 <https://datatracker.ietf.org/doc/html/rfc4291.html#section-2.2>`_
+followed by */<prefix-length>*. `[RFC 4291#section-2.3] <https://datatracker.ietf.org/doc/html/rfc4291.html#section-2.3>`_ 
```

Comment:
Nit: does this make `[]` part of the link test? We probably don't want that. Later there is a similar occurrence for `()`.

Reply:
Yeah the reason I did that one in brackets is to represent where the source is for the whole sentence is. The one before that is part of the sentence itself so it doesn’t have brackets. I can change it however is needed though

Reply:
Fixed. Standardized. No parenthesis or brackets within sentences. If referring to whole sentences, added with brackets after sentence completion with brackets.